### PR TITLE
fix: QA of Docstrings.

### PR DESCRIPTION
**Pull request by shani-leroux**
### Description

- [ ] I've included a concise description of what each node does

### Styleguide

- [ ] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [ ] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

### Testing

- [ ] This PR includes a unit test ([example here](https://github.com/flojoy-io/nodes/blob/c28808f2d0aec8f116cdb3fccb46c4f5256574e9/TRANSFORMERS/ARITHMETIC/ADD/ADD_test_.py) and/or ideally a screenshot of the node's output on an example app.

### CODE REVIEW

File: AI_ML/IMAGE_CLASSIFICATION/HUGGING_FACE_PIPELINE/HUGGING_FACE_PIPELINE.py
Comment (jjerphan) [inline AI_ML/IMAGE_CLASSIFICATION/HUGGING_FACE_PIPELINE/HUGGING_FACE_PIPELINE.py:1]: I came up with a minimal docstring when originally working on it.

Thank you for improving it!

**Review by jjerphan — APPROVED** [review]
Thank you, @shani-leroux.

**Comment by jalium** [comment]
Hey @shani-leroux - what's the status of this PR? Is it just awaiting conflict resolution?

**Review by JeffDotPng — APPROVED** [review]
lgtm

**Pull request merged**